plexpy/datafactory.py

```diff
@@ -1225,6 +1243,18 @@ def get_user_stats(self, rating_key=None, grouping=None):
         
         group_by = 'session_history.reference_id' if grouping else 'session_history.id'
 
+        rating_keys = []
+        if media_type == 'collection':
+            pms_connect = pmsconnect.PmsConnect()
+            result = pms_connect.get_item_children(rating_key=rating_key, media_type=media_type)
+
+            for child in result['children_list']:
+                rating_keys.append(child['rating_key'])
+        else:
+            rating_keys.append(rating_key)
```

Comment:
List comprehension is slightly faster.
```suggestion
        if media_type == 'collection':
            pms_connect = pmsconnect.PmsConnect()
            result = pms_connect.get_item_children(rating_key=rating_key, media_type=media_type)
            rating_keys = [child['rating_key'] for child in result['children_list']
        else:
            rating_keys = [rating_key]
```

Reply:
Cool way.